functions/Handlers.js

```diff
@@ -20,6 +20,17 @@ class RequestError extends Error {
 }
 
 const COLLECTION_NAME_ORDERS = 'orders';
+const CUSTOMER_CAPABILITIES = ['publish', 'subscribe', 'history', 'presence'];
+const RIDER_CAPABILITIES = ['publish', 'subscribe'];
+
+const checkAblyApiKey = (key) => {
```

Comment:
I think this logic around getting the env vars is a bit confusing:
- For me, the name 'checkAblyApiKey' implies that this function is just a side effect, so it feels odd that it also returns the key.
- If the environment variable exists it will be a string, so it's unnecessary to actually check the runtime type rather than just check for existence.
- I don't understand the value of creating the `ablyApiKeyForRiders` and `ablyApiKeyForCustomers` functions?

Reply:
This, perhaps, interacts with your [other observation](https://github.com/ably/asset-tracking-backend-demo/pull/7#discussion_r916645293).

My intention was to reduce repetition and make the code fluid, which I accept may have backfired to making it less readable. 🤔 

I'll revisit this area.